go.mod

```diff
@@ -195,6 +195,17 @@ require (
 	modernc.org/token v1.0.0 // indirect
 )
 
+require (
+	github.com/liamg/clinch v1.5.6
+	github.com/liamg/tablewriter v0.0.6-0.20220511154856-265b7cfd7979
```

Comment:
Do you have a plan to send a PR upstream? I know it is not so active now and the PR may not be merged. But I still prefer using upstream as much as possible since it is hard for folk to follow upstream changes. I also added some commits.
https://github.com/olekukonko/tablewriter/commit/2519d3e28fd6146db290cc237cbd6eff734dbc81

Reply:
Yes, I will try to get something sorted today. I had to break a couple of features that we are not using to get it working, so I'll sort that out and make the use of unicode characters etc. configurable first.

Reply:
It looks like a lot of PRs have gone unnoticed for a while so it's tempting to start a small project that does exactly what we need with no extras...